testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -202,7 +202,9 @@ private enum ExtensionConf {
         )),
         WILDFLY_27_0("WildFly27.0", WILDFLY_26_0, Arrays.asList(
                 "org.wildfly.extension.clustering.ejb",
-                "org.wildfly.extension.datasources-agroal"
+                "org.wildfly.extension.datasources-agroal",
```

Comment:
@yersan is this the correct place where to put new extensions?

Reply:
We already discussed this point via chat and it's now fixed.